test_threadleak.py

```diff
@@ -32,11 +32,18 @@ def test_leak(self):
 threadleak = True
 """
 
+INI_ENABLED_WITH_EXCLUDE = """
+[pytest]
+threadleak = True
+threadleak_exclude = leaked-thread
```

Comment:
Better use “leaked-thread-1” so we exclude the first but detect the second thread. Otherwise the test will pass if the code wrongly exclude all threads.